docs/docs/genai/mcp/index.mdx

```diff
@@ -0,0 +1,239 @@
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+# MLflow MCP Server
+
+:::info
+- This feature is experimental and may change in future releases.
+- MLflow 3.4 or newer is required.
+:::
+
+The MLflow Model Context Protocol (MCP) server enables AI applications and coding assistants to interact with MLflow traces programmatically. MCP is an open protocol that provides a standardized way for AI tools like Claude, VS Code extensions, and other language models to access external data sources and tools.
+
+The MLflow MCP server exposes all MLflow trace management operations through the MCP protocol, allowing AI assistants to:
+
+- Search and retrieve trace data
+- Analyze trace performance and behavior
+- Log feedback and assessments
+- Manage trace tags and metadata
+- Delete traces and assessments
+
+This integration makes it easy to incorporate MLflow tracing capabilities into AI-powered development workflows, enabling more intelligent analysis and management of your GenAI applications.
+
+## Prerequisites
+
+- MLflow version 3.4 or newer
+- An MCP-compatible client (VS Code, Cursor, Claude, etc.)
+
+## Set up
+
+Configure the MLflow MCP server in your MCP client by adding the server configuration to your client's settings file:
+
+<Tabs>
+  <TabItem label="VS Code" value="vscode">
+
+Add to your VS Code configuration file (`.vscode/mcp.json`):
+
+```json
+{
+  "servers": {
+    "mlflow-mcp": {
+      "command": "uv",
+      "args": ["run", "--with", "mlflow>=3.4", "mlflow", "mcp", "run"],
+      "env": {
+        "MLFLOW_TRACKING_URI": "<MLFLOW_TRACKING_URI>"
+      }
+    }
+  }
+}
+```
+
+  </TabItem>
+  <TabItem label="Cursor" value="cursor">
+
+Add to your Cursor configuration file (`.cursor/mcp.json`):
+
+```json
+{
+  "mcpServers": {
+    "mlflow-mcp": {
+      "command": "uv",
+      "args": ["run", "--with", "mlflow>=3.4", "mlflow", "mcp", "run"],
+      "env": {
+        "MLFLOW_TRACKING_URI": "<MLFLOW_TRACKING_URI>"
+      }
+    }
+  }
+}
+```
+
+  </TabItem>
+  <TabItem label="Claude" value="claude">
+
+Add to your Claude `.claude/settings.json`:
+
+```json
+{
+  "mcpServers": {
+    "mlflow-mcp": {
+      "command": "uv",
+      "args": ["run", "--with", "mlflow>=3.4", "mlflow", "mcp", "run"],
+      "env": {
+        "MLFLOW_TRACKING_URI": "<MLFLOW_TRACKING_URI>"
+      }
+    }
+  }
+}
+```
+
+  </TabItem>
+</Tabs>
+
+Replace `<MLFLOW_TRACKING_URI>` with your MLflow tracking server URL:
+
+- **Local server**: `http://localhost:5000`
+- **Remote server**: `https://your-mlflow-server.com`
+- **Databricks**: Use your Databricks workspace URL and set appropriate authentication
```

Comment:
@harupy Can we be more specific here and also link to the relevant databricks mlflow setup guide?